lv_demo.h

```diff
@@ -13,7 +13,7 @@ extern "C" {
 /*********************
  *      INCLUDES
  *********************/
-#if defined(LV_LVGL_H_INCLUDE_SIMPLE) || defined( ARDUINO )
+#if defined(LV_LVGL_H_INCLUDE_SIMPLE)
 #include "lvgl.h"
```

Comment:
I was thinking about `LV_LVGL_H_INCLUDE_SIMPLE`. Instead of adding the `__has include` stuff in `lv_conf_internal.h` it should be added here, in `lv_demo.h`.

Please test if it works well if you add this here:
```c
#if defined __has_include
#  if __has_include("lvgl.h")
#   ifndef LV_LVGL_H_INCLUDE_SIMPLE
#    define LV_LVGL_H_INCLUDE_SIMPLE
#   endif
#  endif
#endif
```